test/util/generator_helpers.rb

```diff
@@ -146,5 +144,15 @@ def generate
                        fixture_info['options'],
                        fixture_info['spec'])
     end
+
+    # Validates multi-platform specs
+    get_platforms(0).zip(
```

Comment:
No, `shuffle` is just a randomize operation.  I think you're getting at the fact that each time we run `rake generate:fixtures` we'll get a bunch of unnecessary changes in these files?

I can change this to be something deterministic so we don't have that if that's a concern.